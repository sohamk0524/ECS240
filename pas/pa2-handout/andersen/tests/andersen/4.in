c Test case with address-of and copy statements
p 7 9
s 0 1 0 2
s 0 1 0 5
s 0 2 0 3
s 0 2 -1 2
s 0 3 0 4
s 0 4 0 2
s 0 4 -1 6
s 0 5 0 3
s 0 5 -1 7
