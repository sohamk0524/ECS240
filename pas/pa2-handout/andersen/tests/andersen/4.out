pt 1 2
pt 1 6
pt 1 7
pt 2 2
pt 2 6
pt 3 2
pt 3 6
pt 4 2
pt 4 6
pt 5 2
pt 5 6
pt 5 7
